in_points: "/camera/depth_registered/points"

out_ground: "/ground_pointcloud"
out_steps:  "/steps_pointcloud"
out_steps_flat: "/steps_flat"
out_steps_tranform: "/steps_tranform"
out_steps_world_flat: "/steps_world_flat"

steps_frame: "/camera_color_optical_frame"

# RANSAC
downsample_size: 0.003          # Downsample leaf voxel size. Higher = greater downsampling
ransac_dt: 0.008                # RANSAC distance threshold. Higher = more inclusive

# Octomap
out_steps_octomap: "/steps_octomap"
out_steps_octomap_flat: "/steps_octomap_flat"
octomap_frame: "/world"
octomap_resolution: 0.025
out_bounding_box: "/bounding_box"
bounding_box: [0, 3, -0.5, 0.5, -0.2, 0.2]  # min_x, max_x, min_y, max_y, min_z, max_z
octomap_threshold: 2                        # number of pointclouds to combine for one octomap